web/src/admin/brands/BrandForm.ts

```diff
@@ -59,7 +59,10 @@ export class BrandForm extends ModelForm<Brand, string> {
                 <input
                     type="text"
                     value="${first(this.instance?.domain, window.location.host)}"
-                    class="pf-c-form-control"
+                    class="pf-c-form-control pf-m-monospace"
+                    autocomplete="off"
+                    spellcheck="false"
+                    inputmode="url"
```

Comment:
Opted for `inputmode` over setting the type to "url". This will hint to the browser that a URL-oriented virtual keyboard is preferred without requiring a valid protocol prefixing the domain.